source/georef/georef.cpp

```diff
@@ -169,7 +169,16 @@ nt::GeographicalCoord Way::nearest_coord(const int number, const Graph& graph) c
 nt::MultiLineString Way::make_multiline(const Graph& graph) const {
     nt::MultiLineString multiline;
     for (auto edge: this->edges) {
-        multiline.push_back({graph[edge.first].coord, graph[edge.second].coord});
+        Edge e = graph[boost::edge(edge.first, edge.second, graph).first];
+        /*
+            Check if the edge way is the same that the current way.
+            Since boost::edge return only one edge between two vertices, if there is parallel edges, we aren't
+            sure it's the right one, and the geom_idx could be out of bounds.
+        */
```

Comment:
If you prefer, to handle parallel edges, you can loop on `boost::out_egdes` as in nearest_edge to find the shape.